ts/components/core/accordion/RawAccordion.tsx

```diff
@@ -0,0 +1,125 @@
+import { View } from "native-base";
+import * as React from "react";
+import { useEffect, useRef, useState } from "react";
+import {
+  AccessibilityProps,
+  Animated,
+  Easing,
+  LayoutAnimation,
+  Platform,
+  StyleSheet,
+  TouchableOpacity,
+  UIManager
+} from "react-native";
+import I18n from "../../../i18n";
+import themeVariables from "../../../theme/variables";
+import customVariables from "../../../theme/variables";
+import IconFont from "../../ui/IconFont";
+import { IOStyles } from "../variables/IOStyles";
+
+// TODO: handle external initial open/closed state
+type Props = {
+  // The header component, an arrow indicating the open/closed state will be added on the right
+  header: React.ReactElement;
+  // The accordion component must accept one children
+  children: React.ReactElement;
+  // The component should be animated?
+  animated?: boolean;
+  headerStyle?: React.ComponentProps<typeof View>["style"];
+  accessibilityLabel?: AccessibilityProps["accessibilityLabel"];
+};
+
+const styles = StyleSheet.create({
+  headerIcon: {
+    alignSelf: "center"
+  },
+  row: {
+    ...IOStyles.row,
+    justifyContent: "space-between"
+  },
+  internalHeader: {
+    flex: 1,
+    paddingRight: themeVariables.contentPadding
+  }
+});
+
+/**
+ * Obtains the degree starting from the open state
+ * @param isOpen
+ */
+const getDegree = (isOpen: boolean) => (isOpen ? "-90deg" : "-270deg");
+
+/**
+ * The base accordion component, implements the opening and closing logic for viewing the children
+ * @param props
+ * @constructor
+ */
+export const RawAccordion: React.FunctionComponent<Props> = props => {
+  const [isOpen, setOpen] = useState<boolean>(false);
+  const animatedController = useRef(new Animated.Value(1)).current;
+  const shouldAnimate = props.animated ?? true;
+  const headerStyle = props.headerStyle ?? {};
+  const accessibilityLabel = props.accessibilityLabel
+    ? `${props.accessibilityLabel}, `
+    : "";
+
+  const arrowAngle = shouldAnimate
+    ? animatedController.interpolate({
+        inputRange: [0, 1],
+        outputRange: ["-90deg", "-270deg"]
+      })
+    : getDegree(isOpen);
+
+  useEffect(() => {
+    if (Platform.OS === "android") {
```

Comment:
```suggestion
    if (isAndroid) {
```
The same but computed only once per running

Reply:
Changed in 9e9dac49ae8d5293fb8605df5ba0ba3ee16a2aee